configs/commercial.js

```diff
@@ -1,10 +1,16 @@
-const path = require('path');
+import licenseHeaderPlugin from 'eslint-plugin-license-header';
 
-module.exports = {
-  plugins: [
-    'license-header'
-  ],
-  rules: {
-    'license-header/header': [2, path.join(__dirname, '/../resources/commercial-license-header.js') ]
+import { fileURLToPath } from 'node:url';
+
+const path = fileURLToPath(new URL('../resources/commercial-license-header.txt', import.meta.url));
```

Comment:
I'm not sure why this is a `.txt` file, as it is (must be) a valid JavaScript snippet.

Reply:
Done.